src/main/java/hudson/plugins/gradle/GradleInstallationProvider.java

```diff
@@ -0,0 +1,15 @@
+package hudson.plugins.gradle;
+
+/**
+ * Interface for providing {@link GradleInstallation} independent of build type.
+ * 
+ * @author Sönke Küper
```

Comment:
Let's not use `@autor` tags. Git should be enough.

Reply:
Okay, i've just done it like in the other files around.. ;)